diktat-ruleset/src/main/kotlin/org/cqfn/diktat/DiktatProcessCommand.kt

```diff
@@ -0,0 +1,102 @@
+package org.cqfn.diktat
+
+import org.cqfn.diktat.api.DiktatCallback
+import org.cqfn.diktat.ktlint.wrap
+import org.cqfn.diktat.ruleset.rules.DiktatRuleSetProvider
+import com.pinterest.ktlint.core.KtLint
+import com.pinterest.ktlint.core.api.EditorConfigDefaults
+import com.pinterest.ktlint.core.api.EditorConfigOverride
+import java.io.File
+
+/**
+ * Command to run `diktat`
+ */
+class DiktatProcessCommand private constructor(
+    val file: File,
```

Comment:
Can you switch to using `java.nio.file.Path` instead?

```suggestion
    val file: Path,
```